tests/monotouch-test/CoreGraphics/CGImagePropertiesGPSTest.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using Foundation;
+#if MONOMAC
+using AppKit;
+#else
+using UIKit;
+#endif
+using CoreGraphics;
+using NUnit.Framework;
+using System.IO;
+
+namespace monotouchtest.CoreGraphics
+{
+	[TestFixture]
+	public class CGImagePropertiesGPSTest
+	{
+		[Test]
+		public void LongitudeRefAndLattitudeRefTest ()
+		{
+			var keys = new NSString [] {
+				new NSString ("LatitudeRef"),
+				new NSString ("Latitude"),
+				new NSString ("LongitudeRef"),
+				new NSString ("Longitude")
+			};
+			var values = new object[] {
+				new NSString ("N"),
+				47.6422f,
+				new NSString ("W"),
+				-122.1367f
+			};
+			CGImagePropertiesGps gps = new (NSDictionary.FromObjectsAndKeys (values, keys));
```

Comment:
I think this would be more interesting if you loaded an actual image file and got the properties that way, because that way you're asserting that iOS is putting the properties in the dictionary in the way we're reading them (as opposed to creating a dictionary with the expected output).

I think this should work for how to load an image:

https://github.com/xamarin/xamarin-macios/blob/9dcd47763e2e64eecf74ec6b293f33e1c0e1632f/tests/monotouch-test/CoreImage/ImageTest.cs#L66-L68

and then the GPS properties should be something like:

```cs
var gps = ci.Properties.Gps;
```

Note that that particular image might not have GPS data, so you might have to add a new image to the test project.